torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -1138,6 +1140,10 @@ XLAGraphExecutor::CompilationResult XLAGraphExecutor::Compile(
   }
 
   xla::XlaComputation computation = ConsumeValue(lowering_ctx.BuildXla());
+  mlir::MLIRContext context;
+  mlir::OwningModuleRef module = mlir::ModuleOp::create(mlir::UnknownLoc::get(&context));
+  HloModuleProto proto(computation.proto());
+  TF_RETURN_IF_ERROR(ConvertHloToMlirHlo(module.get(), &proto, /*import_all_computations=*/false));
```

Comment:
hmm this is not roundtrip right? You just convert HLO to MHLO

Reply:
Not round trip. Just want to solve compiling issue first. Open the draft PR to keep track of progress and allow more people with the work

Reply:
+1

We also want to understand the latency of these translations to StableHLO ABD back from StableHLO

Reply:
I expect the two extra latencies end up being asymmetric - but tolerable